### DISCUSSION TITLE
Is it possible to handle gamepad controls in web brosers running from web assembly?

### DISCUSSION THREAD

**Discussion by AgustinRamiroDiaz** [Leafwing-Studios/leafwing-input-manager#440] [Q&A] [answered]
Thanks so much for this awesome plugin!

I'm working on https://github.com/AgustinRamiroDiaz/snake_bevy and I'm able to hook up the gamepad to my players, and it runs perfectly when running the binary through `cargo run`

I also have the game deployed with `trunk build` at https://agustinramirodiaz.github.io/snake_bevy , but my gamepad is not being caught by the game in this environment. Same behavior when running `trunk serve` locally

Is there something I'm missing or should this be a feature request?

**Comment by alice-i-cecile** [comment, marked as answer]
This is ultimately a question for Bevy itself (and possible gilrs further upstream) :) This plugin just abstracts over the gamepad events received from there.

**Reply by AgustinRamiroDiaz** [reply]
Thanks so much for your time and pro-activeness @alice-i-cecile :heart:

**Reply by AgustinRamiroDiaz** [reply]
I've been out for a while, but now I've updated to bevy 0.15.1 and it's working without any changes :smile:
